docs/content/platform/blocks/todoist/projects.md

```
@@ -0,0 +1,197 @@
+
+### How it works
+<!-- MANUAL: how_it_works -->
+Takes project details and creates via Todoist API.
+<!-- END MANUAL -->
+
+### Inputs
+| Input | Description | Type | Required |
+|-------|-------------|------|----------|
+| name | Name of the project | str | Yes |
+| parent_id | Parent project ID | str | No |
+| color | Color of the project icon | "berry_red" | "red" | "orange" | No |
```

markdownlint-cli2 (0.18.1)

Table column count
Expected: 4; Actual: 6; Too many cells, extra data will be missing

(MD056, table-column-count)